src/components/filterPatterns/modal/OpenFilterButton.tsx

```diff
@@ -43,48 +44,76 @@ export const OpenFilterButton: FC<Props> = ({
   isDisabled = false,
   paddingX = 0,
   backgroundColor = 'unset',
+  showResetButton = true,
+  resetButtonAriaLabel = 'Reset filter',
+  onResetFilter,
 }) => {
+  const shouldDisplayResetButton =
+    showResetButton && isApplied && onResetFilter;
   return (
-    <ChakraButton
-      onClick={onClick}
-      rightIcon={
-        <ChevronRightSmallIcon color={isDisabled ? 'gray.300' : 'gray.500'} />
-      }
+    <Box
       display="flex"
-      justifyContent="space-between"
-      w="full"
+      alignItems="center"
       h={height[variant]}
       paddingX={paddingX}
-      paddingY={paddingY[variant]}
-      isDisabled={isDisabled}
-      cursor={isDisabled ? 'not-allowed' : 'pointer'}
-      color={isDisabled ? 'gray.300' : 'gray.900'}
       backgroundColor={backgroundColor}
     >
-      <chakra.span
-        display="flex"
-        justifyContent="space-between"
-        w="full"
+      <ChakraButton
+        flex="1"
         minW="0"
+        h="full"
+        paddingX={0}
+        isDisabled={isDisabled}
+        cursor={isDisabled ? 'not-allowed' : 'pointer'}
+        color={isDisabled ? 'gray.300' : 'gray.900'}
+        rightIcon={
+          shouldDisplayResetButton ? undefined : (
+            <ChevronRightSmallIcon
+              color={isDisabled ? 'gray.300' : 'gray.500'}
+            />
+          )
+        }
+        onClick={onClick}
       >
         <chakra.span
-          mr="2xl"
-          whiteSpace="nowrap"
           display="flex"
-          alignItems="center"
-        >
-          {label}
-          {Icon ? <Icon h="xs" w="xs" ml="xs" /> : null}
-        </chakra.span>
-        <chakra.span
-          fontWeight="bold"
-          overflow="hidden"
-          textOverflow="ellipsis"
-          whiteSpace="nowrap"
+          justifyContent="space-between"
+          w="full"
+          minW="0"
         >
-          {displayValue && isApplied ? displayValue : null}
+          <chakra.span
+            mr="2xl"
+            whiteSpace="nowrap"
+            display="flex"
+            alignItems="center"
+            flexShrink={0}
+          >
+            {label}
+            {Icon ? <Icon h="xs" w="xs" ml="xs" /> : null}
+          </chakra.span>
+          <chakra.span
+            fontWeight="bold"
+            overflow="hidden"
+            textOverflow="ellipsis"
+            whiteSpace="nowrap"
+            minW="0"
+            flex="1"
+            textAlign="right"
+          >
+            {displayValue && isApplied ? displayValue : null}
+          </chakra.span>
         </chakra.span>
-      </chakra.span>
-    </ChakraButton>
+      </ChakraButton>
+      {shouldDisplayResetButton ? (
```

Comment:
<img width="527" height="236" alt="Screenshot 2025-11-11 at 08 21 45" src="https://github.com/user-attachments/assets/5814fceb-1e6d-4deb-94a2-8c4520dd1094" />

this change is also applied to the homepage version. I don't think this is wanted